Misc/NEWS.d/next/Library/2020-01-12-18-17-00.bpo-39313.DCTsnm.rst

```diff
@@ -0,0 +1,2 @@
+Add a new ``exec_function`` option to ``RefactoringTool`` for making ``exec`` a
```

Comment:
I just noticed that this should be

```suggestion
Add a new *exec-function* option to ``RefactoringTool`` for making ``exec`` a
```

Reply:
I dont think it should be. `--exec-function` added to `main` script but `exec_function` option added to `RefactoringTool`

Reply:
> I dont think it should be. `--exec-function` added to `main` script but `exec_function` option added to `RefactoringTool`

Right. Then maybe we should include also the new option here so users now is available:

```
Add a new ``exec_function`` option (*--exec-function* in the CLI) to ``RefactoringTool`` for making ``exec`` a function. Patch by Batuhan Taskaya.
```

Reply:
Sure!